bin/compile-worker.php

```
@@ -9,18 +9,17 @@
+if ($argc !== 6) {
+    echo 'usage: compile-worker.php <appName> <context> <appDir> <writeDir> <prepend:0|1>' . PHP_EOL;
     exit(1);
 }
 
-[, $appName, $context, $appDir, $tmpDir, $logDir, $prepend] = $argv;
+[, $appName, $context, $appDir, $writeDir, $prepend] = $argv;
 require $appDir . '/vendor/autoload.php';
 
-$compiler = new Compiler($appName, $context, $appDir, $prepend === '1', $tmpDir, $logDir);
-exit($compiler());
+exit((new Compiler($appName, $context, $appDir, $writeDir === '' ? null : $writeDir, $prepend === '1'))());
```

PHPStan (2.2.7)

[error] Parameter `#1` $appName of class BEAR\Package\Compiler constructor expects non-empty-string, string given.


(argument.type)